pkg/app/app.go

```diff
@@ -56,8 +56,7 @@ func (a *WalletApp) BeforeClose(ctx context.Context) bool {
 		return false
 	}
 
-	// Send a cancel message to the prompt and do NOT shutdown
-	a.CtrlChan <- Cancel
```

Comment:
this message in the CtrlChan was never read, so when opening the window again, the chan was already filed, prevent use to do anything

Reply:
I don't think its that simple.
If a wail window is open, there is chances that a channel is waiting for a password or a cancel message.
I think you are addressing the case where wails is open but nothing is waiting because the prompt has already timeout.
So i think we have to consider the 2 cases, because here you are breaking the first one